dll/win32/shell32/CDefView.cpp

```diff
@@ -620,6 +628,20 @@ void CDefView::UpdateListColors()
             m_ListView.SetExtendedListViewStyle(0, LVS_EX_TRANSPARENTSHADOWTEXT);
         }
     }
+    else
+    {
+        SFVM_CUSTOMVIEWINFO_DATA data;
+        data.cbSize = sizeof(data);
+
+        if (S_OK == _DoFolderViewCB(SFVM_GET_CUSTOMVIEWINFO, 0, (LPARAM)&data))
```

Comment:
Does it always need to be `S_OK` ?
Otherwise, `SUCCEEDED(...)`

Reply:
OK.